src/assets/locales/en/translation.json

```diff
@@ -539,7 +539,13 @@
     "repay": "Repay",
     "lend": "Lend",
     "my_lend_positions": "My Lend Positions",
-    "my_borrow_positions": "My Borrow Positions"
+    "my_borrow_positions": "My Borrow Positions",
+    "form_fields": {
+      "lend_amount": "lend amount",
+      "withdraw_amount": "withdraw amount",
+      "borrow_amount": "borrow amount",
+      "repay_amount": "repay amount"
```

Comment:
I checked the full error message and it goes like
`Please enter your lend / withdraw amount`, I think that's not grammatically correct, we could use something like `Please enter the amount to be lent/withdrawn/borrowed/repaid`